src/Tickets/Commerce/Cart.php

```diff
@@ -644,4 +644,15 @@ public function process( array $data = [] ) {
 	public function get_cart_total() {
 		return $this->get_repository()->get_cart_total();
 	}
+
+	/**
+	 * Get the subtotal of the cart.
+	 *
+	 * @since TBD
+	 *
+	 * @return float
+	 */
+	public function get_cart_subtotal() {
```

Comment:
This is a good change but i noticed we are also changing the return type of the abstract_cart which i consider dangerous since this class is probably extended elsewhere!